apps/admin-dashboard/app/shared/constants.ts

```diff
@@ -41,6 +41,7 @@ const ROUTES = [
   '/schools',
   '/schools/create',
   '/schools/:id/edit',
+  '/schools/chapters/:id/create',
```

Comment:
Let's make this 
`/schools/:id/chapter/create`
since we'll be taking the school id and then we'll create a chapter (rather than chapters)